TPI/app/src/main/java/biz/stratadigm/tpi/ui/adapter/VenueAdapter.java

```diff
@@ -48,70 +49,119 @@ public Holder(View view) {
+        try {
+        if (holder instanceof VenueViewHolder) {
+            ((VenueViewHolder) holder).bind(venues.get(position));
+        }
+        } catch (Exception e) {
+            Log.v(TAG, e.toString());
+        }
+                // animate(holder);
+                /*final VenueDTO venue = mDataset.get(position);
+                holder.id.setText("ID: " + venue.getId());
+                holder.name.setText("Name: " + venue.getName());
+                Log.v(TAG, venue.getName());
+                holder.submitted.setText("Submitted: " + venue.getSubmitted());
+                Log.v(TAG, venue.getSubmitted());
+                holder.lat.setText("Lat: " + Float.toString(venue.getLoc().getLat()));
+                holder.lng.setText("Lng: " + Float.toString(venue.getLoc().getLng()));
+                holder.thalis.setText("Thalis: " + venue.getThalis());
+
+                holder.mRoot.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View view) {
+                        SharedPreferences.Editor editor = sharedPreferences.edit();
+                        editor.putString("venue", Long.toString(venue.getId()));
+                        ThaliFragment.venue.setText(Long.toString(venue.getId()));
+                        editor.apply();
+                        MainActivity.jobViewPager.setCurrentItem(2);
+                    }
+                });
+                holder.venueList.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View view) {
+                        Intent i = new Intent(mContext.getApplicationContext(), VenueListThali.class);
+                        i.putExtra("id", position);
+                        mContext.startActivity(i);
+                    }
+                });*/
 
     }
 
     // Return the size of your dataset (invoked by the layout manager)
     @Override
     public int getItemCount() {
 
-        return mDataset.size();
+        return venues.size();
     }
 
-}
+    public void addVenues(List<VenueVO> newVenues) {
+        this.venues.addAll(newVenues);
+    }
+
+    public void setVenues(List<VenueVO> newVenues) {
+        Log.v(TAG, "VenueAdapter: setVenues" + newVenues.size());
+        this.venues = new ArrayList<>(newVenues);
+        notifyDataSetChanged();
+    }
 
+    static class VenueViewHolder extends BaseViewHolder {
 
+        @BindView(R.id.name)
+        TextView nameTextView;
+
+        @BindView(R.id.lat)
+        TextView latTextView;
+
+        @BindView(R.id.lng)
+        TextView lngTextView;
+
+
+        public VenueViewHolder(ViewGroup parent) {
+            super(parent, R.layout.venue_item);
+        }
+
+        public void bind(VenueVO venue) {
+            Log.v(TAG, "ViewHolder " + venue.getName());
+            try {
+                 nameTextView.setText(venue.getName());
+            } catch (Exception e) {
```

Comment:
That's because you forgot to call Butterknife.bind(this, itemView) in the constructor of view holder